.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,51 @@
+name: Update
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update:
+    strategy:
+      matrix:
+        project:
+          - 'scipp/esssans'
+          - 'scipp/sciline'
+          - 'scipp/scippnexus'
+          - 'scipp/scippneutron'
+          - 'scipp/plopp'
+          - 'scipp/essnmx'
+          - 'scipp/essreduce'
+          - 'scipp/beamlime'
+          - 'scipp/essspectroscopy'
+          - 'scipp/chexus'
+          - 'scipp/ess'
+          - 'scipp/chexus'
+          - 'scipp/essreflectometry'
+          - 'scipp/essdiffraction'
+          - 'scipp/esspolarization'
+          - 'scipp/scippuncertainty'
+          - 'scipp/essimaging'
+
+    name: Update downstream projects
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.PAT }}
+          repository: ${{ matrix.project }}
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.X'
+      - run: pip install copier
+      - run: copier update --skip-answered --vcs-ref=HEAD --conflict=rej
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
```

Comment:
If there is no diff, there shouldn't be any change. But as a copier run will result in updating the hash at the top of the copier file it would always create a PR whenenver there is a merge upstream (maybe it's too noisy!)

Reply:
I thought the copier hash doesn't update if it is up-to-date to the template. Maybe I'm wrong... let me try...

Reply:
I just tried. If it already applied the latest templates, it doesn't update the hash either. 
I guess, if we trigger the job on the change of ``main``, it doesn't really make sense to check if there is any changes though... we can check this later when we want use it manually, or run a cron job instead of on the main updates.

Reply:
> (maybe it's too noisy!)

If it turns out to be too noisy, could one limit this to run, e.g., once per week (like you can do with dependabot)? But for now I think this is not necessary, probably?